### DISCUSSION TITLE
How to toggle the syntax highlight theme

### DISCUSSION THREAD

**Discussion by matthijsjanssens** [adityatelange/hugo-PaperMod#1075] [Q&A] [answered]
How would you switch the code syntax highlight theme when toggling dark/light mode? I've tried adding various snippets of code found online to **layouts/partials/extend_head.html** and generating chroma style css files for dark and light themes but it never seems to work. Any ideas?

Here's some of the things I've been trying:
https://hugo.md/post/theme-aware-syntax-highlighting/

Here's the blog post I'm trying it on and its repo
https://matthijsjanssens.github.io/posts/hello-world/
https://github.com/matthijsjanssens/matthijsjanssens.github.io

**Comment by matthijsjanssens** [comment, marked as answer]
Ah, I've found a PaperMod based blog that implemented what I need and included some instructions

https://github.com/jimangel/jimangel.io#update-code-syntax-css

**Comment by matthijsjanssens** [comment]
@adityatelange Can you share how you've made the giscus theme toggle on your blog when you switch dark/light mode? Looks really nice!

**Reply by adityatelange** [reply]
https://github.com/adityatelange/adityatelange.github.io/discussions/2#discussioncomment-3003309

**Reply by matthijsjanssens** [reply]
Thanks a lot! The _transparent_dark_ giscus theme works really well with PaperMod btw.
